Cargo.toml

```diff
@@ -2,15 +2,15 @@
 name = "cidr-utils"
 version = "0.5.10"
 authors = ["Magic Len <len@magiclen.org>"]
-edition = "2021"
+edition = "2018"
```

Comment:
using edition 2018 strictly improves compatibility without any downsides for this crate in isolation

Reply:
But we are in 2023. How does the compatibility be improved if we already set MSRV?

Reply:
Actually you're right, since the MSRV (1.58) is above the introduction of the 2021 edition (1.56) then it's okay to leave this as 2021.